src/js/media-fullscreen-button.js

```diff
@@ -26,16 +26,22 @@ const exitFullscreenIcon = `<svg aria-hidden="true" viewBox="0 0 26 24">
 const slotTemplate = document.createElement('template');
 slotTemplate.innerHTML = `
   <style>
-  :host([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}]) slot:not([name=exit]) > *, 
+  :host([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}]) slot:not([name=exit]) > *,
   :host([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}]) ::slotted(:not([slot=exit])) {
     display: none !important;
   }
 
   /* Double negative, but safer if display doesn't equal 'block' */
-  :host(:not([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}])) slot:not([name=enter]) > *, 
+  :host(:not([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}])) slot:not([name=enter]) > *,
   :host(:not([${MediaUIAttributes.MEDIA_IS_FULLSCREEN}])) ::slotted(:not([slot=enter])) {
     display: none !important;
   }
+
+  :host(:disabled),
+  :host([aria-disabled]) {
+    cursor: not-allowed;
+    opacity: 60%;
```

Comment:
this one is tricky, since it might make the button not pass the contrast ratio req, but not sure if there's another good way of making it visually disabled. Of course, may not be necessary, if we decide that if `document.fullscreenEnabled === false` means the button shouldn't show up.